scripts/generate_cert.sh

```diff
@@ -14,7 +14,7 @@ sudo apt-get update
 sudo apt-get install python-certbot-nginx
 
 # Get certificates (might need sudo)
-certbot certonly --nginx -d "${DOMAIN}" -d "www.${DOMAIN}" --email "${EMAIL}" --agree-tos --redirect
+sudo certbot certonly --nginx -d "${DOMAIN}" --email "${EMAIL}" --agree-tos --redirect
```

Comment:
This is actually wrong for most - if you don’t issue certs for www as well it leads to all kinds of trouble! Please leave as is, and if you want to add a note about cases when it’s not necessary that would be appropriate.

Reply:
Good morning @vsoch!

This failed for me because I never created a record for `www.${DOMAIN}` in Route 53. If I would have, would this have led to the user being redirected to `https://${DOMAIN}` if he/she typed `https://www.${DOMAIN}` in the browser? Or would this have needed additional Nginx configuration? 

Sorry if this sounds dumb... am quite far outside of my comfort zone here...

Reply:
I think we should provide the instruction for how to do it, and then (as you did) the user can delete it, if not appropriate. I think for most professional sites, www is defined.  How about something like:

````bash
# Get certificates (might need sudo)
certbot certonly --nginx -d "${DOMAIN}" -d "www.${DOMAIN}" --email "${EMAIL}" --agree-tos --redirect
```
or if you haven't defined www in your DNS, you can remove that from the request:

````bash
certbot certonly --nginx -d "${DOMAIN}" --email "${EMAIL}" --agree-tos --redirect
````
Let's also leave the actual sudo out, since it says "may need sudo" and the user can add back in if needed.